### DISCUSSION TITLE
Incorrect system time on Android/termux

### DISCUSSION THREAD

**Discussion by shuen2023** [jotaen/klog#352] [Something else]
![Screenshot_20230513_185924_com termux_edit_129841903426873](https://github.com/jotaen/klog/assets/133501179/94d1b503-2efb-4d40-8af3-b858c9457681)

When I started recording with klog, I found that the time klog fetched was not the current system's time. Help take a look, thanks.

**Comment by jotaen** [comment]
This appears to be an issue with Termux / Android and Go, where the timezone for all Go apps seem to ignore the system time zone and erroneously use UTC instead. See https://github.com/termux/termux-packages/issues/2622 and https://github.com/golang/go/issues/20455 for reference.

Unfortunately, this is not something that can be fixed in klog, but we’ll have to wait until a patch will be published in Termux or Go.

**Comment by shuen2023** [comment]
> 这似乎是 Termux/Android 和 Go 的一个问题，所有 Go 应用程序的时区似乎都忽略了系统时区并错误地使用了 UTC。参考[termux/termux-packages#2622](https://github.com/termux/termux-packages/issues/2622)和[golang/go#20455](https://github.com/golang/go/issues/20455)。
> 
> 不幸的是，这不是可以在 klog 中修复的问题，但我们必须等到 Termux 或 Go 中发布补丁。

thanks
